lib/tasks/search.rake

```diff
@@ -2,61 +2,36 @@ require_relative '../task_helpers'
 
 namespace :search do
   desc 'Indexes content updated in the last N minutes.'
-  task :catchup, [:minutes, :index_name, :batch_size, :sleep_time] => :environment do |t, args|
+  task :catchup, [:minutes, :batch_size] => :environment do |t, args|
```

Comment:
I tried to run this `catchup` task and it blew up with:

```
root@forum:~/cs_comments_service# rake search:catchup
/edx/app/forum/cs_comments_service/lib/tasks/flags.rake:6: warning: already initialized constant ROOT
/edx/app/forum/cs_comments_service/lib/tasks/deep_search.rake:7: warning: previous definition of ROOT was here
/edx/app/forum/cs_comments_service/lib/tasks/kpis.rake:7: warning: already initialized constant ROOT
/edx/app/forum/cs_comments_service/lib/tasks/flags.rake:6: warning: previous definition of ROOT was here
/edx/app/forum/cs_comments_service/models/constants.rb:2: warning: already initialized constant COURSE_ID
/edx/app/forum/cs_comments_service/lib/tasks/db.rake:28: warning: previous definition of COURSE_ID was here
W, [2020-10-08T18:31:14.076480 #647]  WARN -- : Overwriting existing field _id in class User.
W, [2020-10-08T18:31:14.126861 #647]  WARN -- : MONGODB | Unsupported client option 'max_retries'. It will be ignored.
W, [2020-10-08T18:31:14.126964 #647]  WARN -- : MONGODB | Unsupported client option 'retry_interval'. It will be ignored.
W, [2020-10-08T18:31:14.126988 #647]  WARN -- : MONGODB | Unsupported client option 'timeout'. It will be ignored.
rake aborted!
NoMethodError: undefined method `each' for 2020-10-08 18:31:14 +0000:Time
/edx/app/forum/cs_comments_service/lib/task_helpers.rb:74:in `catchup_indices'
/edx/app/forum/cs_comments_service/lib/tasks/search.rake:8:in `block (2 levels) in <top (required)>'
/edx/app/forum/.gem/ruby/2.5.0/gems/rake-12.0.0/exe/rake:27:in `<top (required)>'
Tasks: TOP => search:catchup
(See full trace by running task with --trace)
root@forum:~/cs_comments_service#
```

Reply:
Do you ran every rake command? I ask you this question, because I had the same errors when I try to run them locally, but, as example, when I ran rebuild_indices  command during devstack running everything works properly

Reply:
I ran all the rake commands, yes.  The rest worked fine, but `catchup` threw the above NoMethodError.